core/src/main/java/com/yahoo/oak/UnsafeUtils.java

```diff
@@ -13,9 +13,9 @@
 import java.nio.Buffer;
 import java.nio.ByteBuffer;
 
-final class UnsafeUtils {
+public final class UnsafeUtils {
```

Comment:
Is it instead of PR#159? How PR#159 is related?

Reply:
It is used for decoding/encoding the data to a byte array, needed by Memcached transcoder.

Reply:
I removed the dependency on UnsafeUtils.